test/hello_world.2asm.js

```diff
@@ -13,8 +13,8 @@ function asmFunc(global, env, buffer) {
  var Math_abs = global.Math.abs;
  var Math_clz32 = global.Math.clz32;
  function add(x, y) {
-  x = x | 0;
-  y = y | 0;
+  x = (x | 0)
```

Comment:
These changes are a result of STAT nodes being removed from the code base. I will put them back in to fix this.